tests/snippets.zunit

```diff
@@ -52,7 +52,7 @@
 }
 @test 'tldr-completion::gh-r' {
   artifact="$ZINIT[PLUGINS_DIR]/tldr-completion---gh-r"
-  run zinit for as"completion" from"gh-r" id-as'tldr-completion/gh-r' bpick"completions_zsh" @dbrgn/tealdeer
+  run zinit for as"completion" from"gh-r" id-as'tldr-completion/gh-r' bpick"completions_zsh" pick"completions_zsh" @dbrgn/tealdeer
```

Comment:
My understanding of the need for this is that the compile now fails without this because the compile step wasn't run before in the unit test because it was ran non-interactive and therefore triggering the bug which got fixed in this PR.

Reply:
On main and my branch, this one works:

```zsh

[14:31:37] λ  zinit for as"completion" from"gh-r" id-as"tldr-completion/gh-r" bpick"completions_zsh" mv"completions_zsh -> _tldr" pick"_tldr" @dbrgn/tealdeer || echo bad

Downloading dbrgn/tealdeer… (at label: tldr-completion/gh-r…)
(Requesting `completions_zsh'…)
#################################################################################################################################################################################################################### 100.0%
ziextract: WARNING: didn't recognize the archive type of `completions_zsh'  }(no extraction has been done).%f%b
completions_zsh -> _tldr
Note: Compiling: _tldr… OK.
Installed 0 completions. They are stored in the $INSTALLED_COMPS array.
Skipped installing 1 completions. They are stored in the $SKIPPED_COMPS array.
```

The SKIPPED is because the symlink in ~/.zinit/completion/ seems to not get removed on delete:

```
[14:32:48] λ  zinit delete tldr-completion/gh-r -y

[14:33:56] λ  ls -la ../completions/_tldr
lrwxr-xr-x  1 jankatins  staff  60 May  6 14:30 ../completions/_tldr -> /Users/jankatins/.zinit/plugins/tldr-completion---gh-r/_tldr

[14:34:22] λ  ls -la /Users/jankatins/.zinit/plugins/tldr-completion---gh-r/_tldr
ls: /Users/jankatins/.zinit/plugins/tldr-completion---gh-r/_tldr: No such file or directory

[14:34:37] [1] ✗  zinit delete tldr-completion/gh-r -y
No such (plugin or snippet): tldr-completion/gh-r.
```

-> Will adjust the test but I can also imagine that there is some better way to handle a `as"completion"` which does not install a completion.